apps/asap-server/src/controllers/news-and-events.ts

```diff
@@ -16,6 +16,11 @@ export default class ResearchOutputs {
   }): Promise<ListNewsAndEventsResponse> {
     const { total, items } = await this.newsAndEvents.fetch({
       ...options,
+      filter: {
+        path: 'data.type.iv',
+        op: 'ne',
+        value: 'Training',
+      },
```

Comment:
Why don't we make an `or` here with the data types we want to fetch?

Reply:
News and events doesn't filter by type for now. This filter just ensures we don't see Training entries.